src/update-feeds-loops/submit-transactions.ts

```diff
@@ -82,7 +82,12 @@ export const submitTransaction = async (
         );
 
         logger.debug('Getting nonce.');
-        const nonce = await provider.getTransactionCount(sponsorWallet.address, blockNumber);
+        const goNonce = await go(async () => provider.getTransactionCount(sponsorWallet.address, 'latest'));
```

Comment:
We must use specific blockNumber to not cause any race-conditioning between different RPC providers or Airseeker-v2 instances. By doing [that](https://github.com/api3dao/airseeker-v2/blob/aae424814df3cd44d5aff99212ea1fa4fc0edd82/src/update-feeds-loops/update-feeds-loops.ts#L92), we are ensuring that whole loop is working for the same state on the chain.

Reply:
I think it was because of my carelessness and Copilot...